INSTALL.md

```diff
@@ -183,12 +183,18 @@ bundle config --local build.mysql2 "--with-ldflags=-L/usr/local/opt/openssl/lib
 
 and run `bundle install` again.
 
-#### NPM
+#### Yarn
 
-Run [NPM](https://www.npmjs.com/) to install all the required Node modules:
+Install [Yarn](https://yarnpkg.com/):
 
 ```bash
-npm install
+npm install -g yarn
+```
+
+Run [Yarn](https://yarnpkg.com/) to install all the required packages:
+
+```bash
+yarn install
```

Comment:
There is another task 

```suggestion
yarn install:safe
```

It would be also be nice to update the `./bin/setup` for yarn install and such

https://github.com/3scale/porta/blob/5c4ce627e6b0a3bd79e5ae55b877d627bdea4c06/bin/setup

Normally we should provide one way boot of the application locally:

* Install necessary development libraries depending on the OS
* Run `./bin/setup` which will do minimal steps to have the project running locally i.e. : install gems, install node packages, start the server

Reply:
The Installation guideline is already updated, was part of #1655  https://github.com/3scale/porta/pull/1655/commits/d1608bad472bd975426676859042d94320dbcae9